CSLabs.Api/Controllers/UserController.cs

```diff
@@ -137,5 +137,26 @@ public async Task<IActionResult> ChangePassword(ChangePasswordRequest request)
 
             return Ok();
         }
+
+        [HttpPut("change-role")]
+        public async Task<IActionResult> ChangeRole(ChangeRoleRequest[] request)
```

Comment:
We typically standardize on the List concrete class for request data than arrays.
```suggestion
        public async Task<IActionResult> ChangeRole(List<ChangeRoleRequest> request)
```

Reply:
Pushed a commit for this